account_credit_control/views/account_invoice.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <act_window
+            id="act_account_credit_relation_relation"
+            name="Credit Lines"
+            groups="account_credit_control.group_account_credit_control_manager,account_credit_control.group_account_credit_control_user"
+            domain="[('account_id', '=', active_id)]"
+            res_model="credit.control.line"
+            src_model="account.account"/>
+
+    <record id="invoice_followup_form_view" model="ir.ui.view">
+        <field name="name">invoice.followup.form.view</field>
+        <field name="model">account.invoice</field>
+        <field name="inherit_id" ref="account.invoice_form"/>
+        <field name="arch" type="xml">
+            <notebook position="inside">
+                <page string="Credit Control"
+                      groups="account_credit_control.group_account_credit_control_manager,account_credit_control.group_account_credit_control_user,account_credit_control.group_account_credit_control_info">
+                    <group>
+                        <field name="credit_policy_id" widget="selection"
+                               string="Manual Credit Control Policy"
+                               attrs="{'invisible': [('credit_policy_id', '=', False)]}"
```

Comment:
@grindtildeath Ok, and why this one is readonly and when it is filled ?
cc\ @sylvainvh